packages/flutter_tools/bin/fuchsia_asset_builder.dart

```diff
@@ -38,7 +38,7 @@ Future<void> main(List<String> args) {
 
 Future<void> writeFile(libfs.File outputFile, DevFSContent content) async {
   outputFile.createSync(recursive: true);
-  final List<int> data = await content.contentsAsBytes();
+  final List<int> data = content.contentsAsBytes();
   outputFile.writeAsBytesSync(data);
 }
```

Comment:
this doesn't really need to be async any more...

Reply:
Fixed